events/class_tests/possible_participant_tests.py

```diff
@@ -30,12 +30,12 @@ def get_event():
 
 @pytest.fixture
 def get_possible_event_participants():
-    return [PossibleParticipant.objects.get(id=f'{id}') for id in range(3, 7)]
+    return [PossibleParticipant.objects.get(id=f'{id}') for id in range(3, 8)]
 
 
 @pytest.fixture
 def get_possible_meeting_participants():
-    return [PossibleParticipant.objects.get(id=i) for i in range(4, 6)]
+    return [PossibleParticipant.objects.get(id=i) for i in range(4, 7)]
```

Comment:
I would really love, if possible, if you find any way to use some constants/variables instead of hard coding these numbers.

^Relevant for the two methods^

Reply:
These numbers represent the ids of the possible participants that are in the DB.
In my opinion, there is no need to use constants for the starting and ending range of ids.

Reply:
i agree with @RavidYael, as in my previous comment, I think it is better to name the constants according to what they represent

Reply:
Done 😄